### PR TITLE
For #25435: Handle ERROR_BAD_HSTS_CERT error.

### PR DESCRIPTION
**Pull request by mcarare**
Needs https://github.com/mozilla-mobile/android-components/pull/12236

### Pull Request checklist

- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Screenshots**: This PR includes screenshots or GIFs of the changes made or an explanation of why it does not
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features. In addition, it includes a screenshot of a successful [accessibility scan](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor&hl=en_US) to ensure no new defects are added to the product.

### To download an APK when reviewing a PR:
1. click on Show All Checks,
2. click Details next to "Taskcluster (pull_request)" after it appears and then finishes with a green checkmark,
3. click on the "Fenix - assemble" task, then click "Run Artifacts".
4. the APK links should be on the left side of the screen, named for each CPU architecture

### CODE REVIEW

File: app/src/main/assets/lowMediumErrorPages.js [hunk 1]
```diff
@@ -71,6 +72,24 @@ function updateShowSSL(queryMap) {
     }
 };
 
+/**
+ * Used to show or hide the "advanced" button based for the HSTS error page
+ */
+function updateShowHSTS(queryMap) {
+    /** @type {'true' | 'false'} */
```
Comment (rocketsroger) [inline app/src/main/assets/lowMediumErrorPages.js:79]: Not sure if this line is needed.

**Review by rocketsroger — APPROVED** [review]
Looks good to me except this one line.

**Review by rocketsroger — APPROVED** [review]
🚢

**Comment by jonalmeida** [comment]
Everything works fine for me locally. I'm guessing this is some form of intermittent dependency fetch that didn't work. So I can only think to re-build as a result.

**Pull request merged**